include/zephyr/bluetooth/bluetooth.h

```diff
@@ -235,6 +235,11 @@ int bt_enable(bt_ready_cb_t cb);
  * stored with @kconfig{CONFIG_BT_SETTINGS}. These can be restored
  * with settings_load() before reenabling the stack.
  *
+ * This API does _not_ clear previously registered callbacks
+ * like @ref bt_le_scan_cb_register and @ref bt_conn_cb_register.
+ * That is, the application shall not re-register them when
+ * the Bluetooth subsystem is re-enabled later.
```

Comment:
It's more than just that, isn't it? 

Anything that does not require `bt_enable` to be called first is not affected by this. That include the callbacks as you write, but all any registered GATT services, L2CAP and ISO servers, etc.

Reply:
Yes, there is more to it. I indented to make this generic. Maybe something like 
```suggestion
 * This API does _not_ clear any of the previously registered callbacks,
 * for example callbacks like @ref bt_le_scan_cb_register and @ref bt_conn_cb_register.
 * That is, the application shall not re-register them when
 * the Bluetooth subsystem is re-enabled later.
```
?

Reply:
My point is that it is true for much more than just callbacks. 

It is OK to just keep this text for callbacks, but we will need to expand this to cover all the cases at some point